src/main/java/use_case/list/ListInteractor.java

```diff
@@ -12,7 +12,7 @@
 import entity.User;
 
 /**
- * The list display Interactor.
```

Comment:
I don't think this description change is accurate, since it's still used to display media normally, whether it involves removing media or not.